sway/tree/workspace.c

```diff
@@ -147,6 +147,8 @@ void workspace_begin_destroy(struct sway_workspace *workspace) {
 	if (workspace->output) {
 		workspace_detach(workspace);
 	}
+	wl_signal_emit(&workspace->node.events.postdestroy, &workspace->node);
```

Comment:
Yeah, this is wrong – interferes with the logic in 9350a52c0. Since it doesn't make sense to split the signal here either, I'm going back to just the single destroy signal again.